### DISCUSSION TITLE
Refresh or update CRUD auto-generated screen after database change?

### DISCUSSION THREAD

**Discussion by seaelem** [Budibase/budibase#5115] [Help] [answered]
Hi, I am very new to budibase (literally a couple of hours).  Building a quick test app for the junior netball team I coach.  Created a table structure and auto-generated CRUD interface (baby steps here) all good.  Then decided I did not want one of the fields / columns in the database and so deleted it.  This field had a dropdown box pointing at it in the CRUD auto-generated screen.  Do I. now need to manually delete the dropdown on the CRUD "new record" screen or is there some way to refresh / update it?

See screenshots - the 'Game' table had a column called 'Quarter' which I removed.  Do I now need to remove the dropdown in the game table new record CRUD screen?

Cheers,
Carl.

<img width="960" alt="Screen Shot 2022-03-27 at 11 48 53 am" src="https://user-images.githubusercontent.com/26319178/160261881-f02708c5-074a-48ce-a3f7-cfe143a984c9.png">
<img width="721" alt="Screen Shot 2022-03-27 at 11 49 14 am" src="https://user-images.githubusercontent.com/26319178/160261882-0996a730-b3a5-43a6-a21a-78e78361edd0.png">

**Comment by melohagan** [comment, marked as answer]
Hi Carl,

If you don't remove the Quarter dropdown from your form, nothing will break. You'll still be able to create and edit rows as before, the Quarter dropdown will just be ignored.

As you've shown in the screenshot, deleting a column from a table does not automatically remove it from the screen forms. Potentially you would want this field to be kept, because it could still impact the logic / filters  of other elements.

**Reply by seaelem** [reply]
Ah thanks for that. That’s what I thought but appreciate the answer a great deal thank you.
